scripts/validate-deployment.sh

```diff
@@ -5,19 +5,20 @@ set -euo pipefail
 ## Query Prometheus for the daml_health_status metric and validate that it has the value 1, representing a healthy service
 
 retry=0
-result=
-until [ "$retry" -ge 5 ]; do
+result=''
+
+until [[ "${retry}" -ge 5 ]]; do
   # Query Prometheus for daml_health_status >= 1
-  result=$(curl --retry 10 --retry-connrefused 'http://localhost:9090/api/v1/query?query=daml_health_status+%3E%3D+1')
+  result=$(curl -fsS --retry 5 --retry-connrefused 'http://localhost:9090/api/v1/query?query=daml_health_status+%3E%3D+1')
   # http-json, canton domain and participant
   # trigger-service does not expose health data
-  if [ $(echo "$result"| jq -r '.data.result | length') -eq 3 ]; then
+  if [[ $(echo "${result}" | jq -r '.data.result | length') -eq 3 ]]; then
```

Comment:
What's the benefit here?

Reply:
Same here, explicit Bash interpolation to avoid confusions with `result` from JSON parsing for example